### ISSUE TITLE
Recording cursor jumps back in time

### ISSUE THREAD

**Issue by drautzburg** [muse-sequencer/muse#1282]
This may well not be a bug at all, but an error on my side. Either way, I'd appreciate any help. 

```
Version: 4.2.1
Tag:	 4.2.1-1-gcb0bce73-dirty
```

**This is what happened:**

I had an extended recording session last weekend, where we recorded five songs, with roundabout 10 simultaneous tracks (which did cause a few glitches BTW). All songs were recorded one after the other as part of the same main file. Between recordings I set some markers, so I could locate the beginnings of each song later during mixing.

The first two songs went fine. When I ended the recording of the third song, I noticed, that the marked marker was one pointing to an earlier song, the cursor was at a position way to early and it was overdubbing an earlier song.

With the fourth song, the same thing happened. This time I made extra sure, the recording was at the tail end of the project and the correct marker was marked. I couldn't watch the recording proceed, because I had to play bass. When I ended the recording, I noticed again, that the cursor wasn't where it was supposed to be, but apparently had jumped back in time.

For the fifth song I created a new main file and everything worked smoothly.

**What I considered:**

I am 100% sure, "Loop" wasn't active during the recording. I've made this mistake before.

One thing that was different to previous recording sessions, was that my piano was used. The piano has MIDI connections to and from my computer. So, I suspected, that the "go to left mark" MIDI remote control key was hit accidentally. The next day, I tried to provoke the jumping back with the piano keyboard, but couldn't. Only after I rebooted the computer the MIDI remote control worked again (not MusE's problem). This and the fact, that there was no piano involved in the fourth song, makes this an unlikely explanation.

I have a vague feeling, that the markers had something to do with the problem. Another vague feeling, I had during the recording session was that MusE ran out of "tape". This session was probably the longest recording, I ever did. So MusE rewound the "tape" (probably mythical thinking)

I then checked the .wav files, that MusE writes, and so far, they look fine. It seems, no real harm was done and I am currently mixing one of the songs, by importing the .wav files into a new main file.

**What's next**

I will try to reproduce the situation in a week or so. If I can't do this, we might as well close this bug report.  In the meanwhile I'd be happy to hear any advice or explanation, why this might have happened.

**Comment by drautzburg** [comment]
Today, I tried to reproduce the "looping back". I recorded 5 "songs" with the same number of simultaneous tracks and set markers after each song.

**I could not reproduce the error.**

During the real recording session, I had a handful xruns on the meter and some tracks showed some audible glitches. This was a mystery to me, as I had jackd running with generous buffer sizes. Anyway, this also didn't occur today.

I am closing this issue.
